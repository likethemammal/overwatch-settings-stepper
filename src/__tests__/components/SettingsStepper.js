import React from 'react';
import {shallow} from 'enzyme';

import general from '@likethemammal/general-components'

const shared = general.__tests__._shared.tests

import { Toggle, SettingsStepper } from '../../'


describe('SettingsStepper', () => {
    // Render a checkbox with label in the document
    const component = shallow(
        <SettingsStepper
        />
    )

    shared.SNAPSHOTS_SHOULD_MATCH([
        component
    ])

})