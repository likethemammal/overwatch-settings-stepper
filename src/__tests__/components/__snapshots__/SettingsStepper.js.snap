// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SettingsStepper Snapshot 0 should match 1`] = `undefined`;
